addons/mail/static/src/core/common/special_mentions.js

```diff
@@ -0,0 +1,11 @@
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+
+const specialMentionRegistry = registry.category("discuss.special_mentions");
+
+specialMentionRegistry.add("everyone", {
+    channel_types: ["channel", "group"],
+    label: _t("everyone"),
+    displayName: _t("Everyone"),
```

Comment:
Let's not translate the `label` and `displayName`, so this matches other messaging apps.